.travis.yml

```diff
@@ -45,40 +48,62 @@ addons:
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        wget https://repo.continuum.io/miniconda/Miniconda-latest-MacOSX-x86_64.sh -O miniconda.sh;
       else
-        export TRAVIS_PYTHON_VERSION="3.6";
+        wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
       fi
-    # We do this conditionally because it saves us some downloading if the version is the same.
-    - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-        if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-          wget https://repo.continuum.io/miniconda/Miniconda-latest-MacOSX-x86_64.sh -O miniconda.sh;
-        else
-          wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
-        fi
+    else
+      if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        wget https://repo.continuum.io/miniconda/Miniconda3-4.4.10-MacOSX-x86_64.sh -O miniconda.sh;
       else
-        if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-          wget https://repo.continuum.io/miniconda/Miniconda3-4.4.10-MacOSX-x86_64.sh -O miniconda.sh;
-        else
-          wget https://repo.continuum.io/miniconda/Miniconda3-4.4.10-Linux-x86_64.sh -O miniconda.sh;
-        fi
+        wget https://repo.continuum.io/miniconda/Miniconda3-4.4.10-Linux-x86_64.sh -O miniconda.sh;
       fi
+    fi
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export CONDA_HOME="$HOME/miniconda"
+  - export PATH="$CONDA_HOME/bin:$PATH"
+  - conda create -n ert_env pyqt=4.11.4 --yes
+  - source activate ert_env
+  - pip install --upgrade -r requirements.txt
+  # Checkout libres - need libecl version
+  - source .libres_version
+  - git clone https://github.com/equinor/libres
+  - pushd libres
+  - git checkout tags/$LIBRES_VERSION
+  - source .libecl_version
+  - popd
+  - git clone https://github.com/equinor/libecl
+  - pushd libecl
+  - git checkout tags/$LIBECL_VERSION
+  - popd
 
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export CONDA_HOME="$HOME/miniconda"
-    - export PATH="$CONDA_HOME/bin:$PATH"
-    - hash -r
-    - pip install cwrap
-    - conda config --set always_yes yes --set changeps1 no
-    - conda update -q conda
-    # Useful for debugging any issues with conda
-    - conda info -a
-    - travis/install_python_packages.py
-
+install:
+  # For now we have to make install libecl.
+  # Remove when it's possible to pip install
+  - bash .build_install.sh libecl
+  - bash .build_install.sh libres
 
 script:
-  - travis/build_total.py ert
+  - mkdir build
+  - pushd build
+  - cmake .. -DBUILD_TESTS=ON
+             -DENABLE_PYTHON=ON
+             -DBUILD_APPLICATIONS=ON
+             -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR
+             -DCMAKE_PREFIX_PATH=$INSTALL_DIR
+             -DERT_USE_OPENMP=ON
+             -DCMAKE_F_CLAGS='-Werror=all'
+             -DCMAKE_CXX_FLAGS='-Werror -Wno-unused-result'
+  - make install
+  - ctest --output-on-failure $TEST_SUITE
```

Comment:
I guess `TEST_SUITE` can go?

Reply:
yes, that's wrongly copied. I'll remove it